docs/index.asciidoc

```diff
@@ -24,6 +24,11 @@ The CSV filter takes an event field containing CSV data, parses it,
 and stores it as individual fields with optionally-specified field names.
 This filter can parse data with any separator, not just commas.
 
+[id="plugins-{type}s-{plugin}-ecs_metadata"]
+==== Event Metadata and the Elastic Common Schema (ECS)
+The plugin behaves the same except giving a warning when ECS is enabled and `target` isn't set.
+It is recommended to set the `target` option to avoid potential schema conflicts.
```

Comment:
```suggestion
The plugin behaves the same regardless of ECS compatibility, except giving a warning when ECS is enabled and `target` isn't set.

TIP: Set the `target` option to avoid potential schema conflicts.
```

Reply:
I like the way y'all handled this.  You can use a TIP to put more emphasis on the impact of ECS.  If you want to try this, it will render like this: 

<img width="860" alt="Screen Shot 2021-05-28 at 8 01 55 PM" src="https://user-images.githubusercontent.com/35154725/120051764-f7c67700-bfef-11eb-8e01-f8da6becdf5b.png">